addons/website/models/website.py

```diff
@@ -386,6 +386,10 @@ def configurator_skip(self):
         website = self.get_current_website()
         website.configurator_done = True
 
+    @api.model
+    def configurator_send_iap(self, attempt, final_industry_choice):
+        self._website_api_rpc('/api/website/new_industry', {'attempt': attempt, 'final_industry_choice': final_industry_choice, 'language': self.env.context.get('lang')})
```

Comment:
```suggestion
        self._website_api_rpc('/api/website/new_industry', {'attempt': attempt, 'final_industry_choice': final_industry_choice, 'lang': self.env.context.get('lang')})
```
Nitpicking, you can ignore this is you want, but we usually use `lang` shortcut everywhere.
Minor bonus (just to find one): less data sent on the network, but it's extreme nitpicking